app/scripts/controllers/transactions/index.js

```diff
@@ -1600,6 +1604,9 @@ export default class TransactionController extends EventEmitter {
     sendFlowHistory = [],
     actionId,
     options,
+    ///: BEGIN:ONLY_INCLUDE_IN(blockaid)
+    securityAlertResponse,
```

Comment:
[nit] what if we pass `securityAlertResponse` in options param since it won't be needed for all _createTransaction calls?